src/open_inwoner/templates/pages/profile/categories.html

```diff
@@ -0,0 +1,13 @@
+{% extends 'master.html' %}
+{% load i18n form_tags %}
+
+{% block content %}
+<h1 class="h1" id="title">
```

Comment:
Tiny detail: we would like to slowly get rid of meaningless classes that are not clearly describing the repeating components, and one way to do this is to start using NL-Design-System classes, like "utrecht-heading-1" here. 
(Or if a component is completely new: give it its own heading class, but that's not the case here)

Reply:
I changed it to `utrecht-heading-1`